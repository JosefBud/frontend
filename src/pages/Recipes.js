import React, { Component } from 'react';
import { Container, Row, Col } from 'react-bootstrap';

export default class Recipes extends Component {
  render() {
    return (
      <Container>
        <Row className="text-center">
          <Col>
            <h1>Recipes search page</h1>
          </Col>
        </Row>
      </Container>
    );
  }
}
