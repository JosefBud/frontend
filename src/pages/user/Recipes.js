import React, { Component } from 'react';
import { Link } from '@reach/router';
import { Container, Row, Col, Table } from 'react-bootstrap';

export default class UserRecipes extends Component {
  listRecipes() {}
  render() {
    const listItems = [];

    for (let i = 0; i < 20; i++) {
      listItems.push(
        <tr key={i}>
          <td>
            <Link to="#">Recipe {i}</Link>
          </td>
        </tr>
      );
    }

    return (
      <Container className="text-center">
        <Row>
          <Col>
            <h1>Your Original Recipes</h1>
            <Table striped bordered>
              <tbody>{listItems}</tbody>
            </Table>
          </Col>
          <Col>
            <h1>Your Adapted Recipes</h1>
            <Table striped bordered>
              <tbody>{listItems}</tbody>
            </Table>
          </Col>
        </Row>
      </Container>
    );
  }
}
