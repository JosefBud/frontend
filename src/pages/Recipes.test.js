import React from 'react';
import renderer from 'react-test-renderer';

import Recipes from './Recipes';

describe('<Recipes />', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Recipes />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
