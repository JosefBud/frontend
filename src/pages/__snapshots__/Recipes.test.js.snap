// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Recipes /> renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="text-center row"
  >
    <div
      className="col"
    >
      <h1>
        Recipes search page
      </h1>
    </div>
  </div>
</div>
`;
